### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by Thraganux** [FreeCodeCamp/FreeCodeCamp#6820]
Challenge [Build a Simon Game](http://www.freecodecamp.com/challenges/build-a-simon-game) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64; rv:43.0) Gecko/20100101 Firefox/43.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

This is more of a suggestion, so I'm not sure where to mention it:
Due to the issues i've read with MP3s, making seamless sound loops, lossy formats, and working with HTML5 audio, I wonder if it would be a good idea to either have the files in longer formats or point the user to the Web Audio API (which is a little harder to use).  The instructions could mention that Web Audio API is optional. Even using https://github.com/Hivenfour/SeamlessLoop didn't quite work, due to the .mp3s used.

 It's especially annoying for perfectionists who can hear the tone differences between the audio samples and the simon says example project. After playing with the Web Audio API, I had to look at where the original example was getting the button-sounds from (really cool, btw), and frankly think that the extra work involved in Web Audio API makes a better, more-polished portfolio piece.

**Comment by QuincyLarson** [comment]
@Thraganux is there an easier option than making them use new API? Like having higher quality sounds? Or am I missing the point you're making?

**Comment by Thraganux** [comment]
Sorry, I have been sick.

Well, I am not sure about if there is an easier version than using a new API.  I have done some research and found that the type of files provided (mpe3) usually don't allow for seamless looping sounds without some sort of API (no gaps, stutters, or pauses in the sound as the sound file is played over and over).  Maybe someone figured it out.  I saw someone mention that .ogg files can be looped seamlessly, but I have not tried ti for myself.

In the long run, I actually found that it looked and sounded more professional to use Web Audio.   It's not too hard to figure out and it's a great api to know.  It was also much easier than trying to use the MP3s to play a tone that played seamlessly for as long as the user was holding the mouse down on a button.
